samples/wifi/sta/README.rst

```diff
@@ -0,0 +1,83 @@
+.. _wifi_sta_sample:
+
+Wi-Fi: Sta
+###########
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample allows Nordic Semiconductor's Wi-Fi chipset to connect with Access point.
```

Comment:
Replace with "This sample demonstrates how to connect the Wi-Fi Station to a specified Access Point"